libHVCAENx527App/Db/HVCAENx527ch.tpl

```diff
@@ -634,3 +634,11 @@ alias("$(PSNAME):SIM:$(CHANNUM):pwdnmode","$(PSNAME):SIM:$(CHANNUM):pwdnmode:fbk
 alias("$(PSNAME):SIM:$(CHANNUM):tripint","$(PSNAME):SIM:$(CHANNUM):tripint:fbk")
 alias("$(PSNAME):SIM:$(CHANNUM):tripext","$(PSNAME):SIM:$(CHANNUM):tripext:fbk")
 
+record(calc, "$(PSNAME):$(CHANNUM):intol") {
+    field(INPA, "$(PSNAME):$(CHANNUM):v0set:fbk CP MSS")
+    field(INPB, "$(PSNAME):$(CHANNUM):vmon CP MSS")
+	field(INPC, "$(P):voltage_tolerance CP MSS")
+	field(CALC, "(B>=(1-(C/100))*A) && (B<=(1+(C/100))*A)")
```

Comment:
a comment above this record might be nice to explain what the (complex?) logic in the calc is actually doing